src/main/java/frc/robot/RobotContainer.java

```diff
@@ -256,6 +257,8 @@ private void configureButtonBindings() {
     // x-stance
     oi.getXStanceButton().onTrue(Commands.runOnce(drivetrain::enableXstance, drivetrain));
     oi.getXStanceButton().onFalse(Commands.runOnce(drivetrain::disableXstance, drivetrain));
+    oi.getVisionIsEnabledSwitch().onTrue(Commands.runOnce(vision::visionSubsystemIsEnabledIsTrue()));
```

Comment:
Since the vision subsystem cannot reference the drivetrain subsystem, you will have to run two commands. This can be done with the Commands.parallel factory method. For example:

```
Commands.parallel(Commands.runOnce(vision::disableVision, drivetrain:: resetPoseRotationToGyro));
```

Reply:
done